openfl/interface/aggregator.py

```
@@ -40,6 +42,13 @@ def start_(plan, authorized_cols, secure):
 
     from openfl.federated import Plan
 
+    if is_directory_traversal(plan):
+        echo('Federated learning plan path is out of the openfl workspace scope.')
+        sys.exit(1)
```

Can we raise an exception instead of calling `sys.exit(1)`?

---

Exceptions are expected to be handled or to help get additional information about bug to fix it. But in that case we have already provided information about reason of error and want to exit with error code, that could be handled by system. It would be great if these were different error codes and could be automatically handled, but it looks like another topic for discussion.